roadmap.md

```diff
@@ -1,68 +1,39 @@
-  * [System.Windows.Controls.Ribbon](src/Microsoft.DotNet.Wpf/src/System.Windows.Controls.Ribbon)
-  * [WPF Themes](src/Microsoft.DotNet.Wpf/src/Themes)
-    * [PresentationFramework-Aero](src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Aero)
-    * [PresentationFramework-Aero2](src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Aero2)
-    * [PresentationFramework-AeroLite](src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.AeroLite)
-    * [PresentationFramework-Classic](src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Classic)
-    * [PresentationFramework-Luna](src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Luna)
-    * [PresentationFramework-Royale](src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Royale)
-  * [System.Windows.Presentation](src/Microsoft.DotNet.Wpf/src/System.Windows.Presentation)
-  * [PresentationUI](src/Microsoft.DotNet.Wpf/src/PresentationUI)
-  * [System.Printing](src/Microsoft.DotNet.Wpf/src/System.Printing)
-  * Native Components
-    * [PenIMC_Cor3](src/Microsoft.DotNet.Wpf/src/PenImc)
-    * [WpfGfx_Cor3](src/Microsoft.DotNet.Wpf/src/WpfGfx)
-  
-* Tests:
-  * [DrtXaml](src/Microsoft.DotNet.Wpf/test/DRT/DrtXaml)
-
+# WPF for .NET 5 Roadmap
+WPF is a .NET Core UI framework for building Windows desktop applications. The ownership of WPF was recently transitioned to our team (Developer Ecosystem and Platform team under Windows).This transition enables investments across UI frameworks namely WINUI and WPF to stay aligned and remain future-proof as new technology trends & devices are introduced in the industry.
+
+The roadmap below communicates priorities for evolving and extending the scope of WPF for .NET Core through 2020 and into 2021. It will continue to evolve based on market changes and customer feedback, so please note that the plans outlined here are not exhaustive or guaranteed. We welcome your feedback on the roadmap: please feel free to contribute to existing issues or [file a new issue](https://github.com/dotnet/wpf/issues "file a new issue").
+
+### PROPOSED ROADMAP
+| #  | Milestone  |  Target Delivery |
+| :------------: | :------------: | :------------: |
+|1  |Incorpoating .NET Framework servicing fixes  into .NET Core 3.1 and .NET 5 |Ongoing|
+|2  |Ongoing integration with .NET 5 |20H2|
+|3  |Build out Test Infrastructure to add tests to validate and merge community PRs|20H2|
+|4  |Accessibility updates to .NET 5   |21H1|
+|5  |ARM64 Support|21H1|
+
+### Our SLA
+#### TRIAGING GITHUB ISSUES
+We are committed to a 72 hour (3 working days) turn around on triaging and responding to new issues filed on Github starting July 10th.
+Additionally,  Issues filed prior to July 10th are also being triaged. Issues filed in 2020 will be triaged first based on number of reactions/comments, and we expect to complete doing so by the middle of 20H2. Issues filed before 2020 will be triaged after that based on number of reactions/comments.
```

Comment:
TBH I’m a little concerned about what’s stated here:, especially the last sentence. Many issues that were logged immediately after the repo went live, were logged because they were important to us and thus the first thing we thought of. I get you’re trying to handle almost 500 issues, but basing priority mainly on when it was logged seems problematic to me.

For instance I logged an issue in December 2018 that is around a huge performance regression, and there are also many other great issues logged around that time. I sort of feel the issues we logged right away are getting “punished” for not waiting till July 2020. I’m also a little worried it’ll lead to people closing their issues, and creating new duplicate ones, just to get the visibility they think they deserve. There are [currently 12 regressions logged](https://github.com/dotnet/wpf/issues?q=is%3Aopen+is%3Aissue+label%3Aregression+), and only one of them is from this year, and only barely.

Now I get you further up write "the plans outlined here are not exhaustive", so I'm hoping you'll still at least glance at some of the much older issues and consider to fast track those too,

Reply:
Thanks for the link to the 12 regressions. We will certainly look into them. We are aggressively triaging Issues each week to bring down our Issue count soon. As noted in the roadmap, the plans captured here are not exhaustive, and will evolve in the coming weeks. We will make sure that the roadmap is updated accordingly.